src/ElboDeriv.jl

```diff
@@ -872,12 +874,52 @@ function tile_predicted_image{NumType <: Number}(
   tile_sources = mp.tile_sources[b][tile.hh, tile.ww]
 
   tile_predicted_image(tile,
-          tile_sources,
-          mp,
-          sbs,
-          star_mcs,
-          gal_mcs,
-          accum)
+                       tile_sources,
+                       mp,
+                       sbs,
+                       star_mcs,
+                       gal_mcs,
+                       accum)
+end
+
+
+@doc """
+Calculate the ELBO only for tiles containing a particular source.
+This can save a lot of computation when using autodifferentiation to get
+the Hessian.  The values of the ELBO and derivatives will be incorrect,
```

Comment:
How come the derivatives might not be correct? They'll be correct for one source (the one we've used to select the relevant tiles), right? Does this function compute derivatives for sources other than that one?